src/components/RoomView/Card.tsx

```diff
@@ -160,25 +152,20 @@ const Card = (word: CardObj) => {
     }
   };
 
+  let cardStyles = {};
+  if (word.teamId === team1Id) cardStyles = allCardStyles.redCardStyles;
+  if (word.teamId === team2Id) cardStyles = allCardStyles.blueCardStyles;
+  if (word.teamId === bystanderTeamId) cardStyles = allCardStyles.beigeCardStyles;
+  if (word.teamId === assassinTeamId) cardStyles = allCardStyles.blackCardStyles;
+  if (!word.teamId) cardStyles = allCardStyles.unknownCardStyles;
+
   return (
-    <>
-      {/* if card hasnt been revealed, show this beige version and submit answer on click */}
-      {!word.isVisibleToAll && (
-        <button className="notYetRevealed" value={word.id} onClick={submitAnswer}>
-          {word.wordString}
-        </button>
-      )}
-      {/* if it is visible, show the color for the team, and make it not clickable 
-      (buttons made them more visually appealing for the time being but we can edit css obviously) */}
-      {word.isVisibleToAll && word.teamId === team1Id && <button className="redRevealed">{word.wordString}</button>}
-      {word.isVisibleToAll && word.teamId === team2Id && <button className="blueRevealed">{word.wordString}</button>}
-      {word.isVisibleToAll && word.teamId === bystanderTeamId && (
-        <button className="beigeRevealed">{word.wordString}</button>
-      )}
-      {word.isVisibleToAll && word.teamId === assassinTeamId && (
-        <button className="blackRevealed">{word.wordString}</button>
-      )}
-    </>
+    <div onClick={submitAnswer}>
```

Comment:
I had to wrap the ReactCardFlip in a div component because I was not allowed to put an onClick on `<ReactCardFlip>`